jax/_src/lax/control_flow/loops.py

```diff
@@ -139,7 +140,7 @@ def scan(f, init, xs, length=None):
         ys.append(y)
       return carry, np.stack(ys)
 
-  Unlike that Python version, both ``a`` and ``b`` may be arbitrary pytree
+  Unlike that Python version, both ``xs`` and ``ys`` may be arbitrary pytree
   types, and so multiple arrays can be scanned over at once and produce multiple
```

Comment:
To be consistent, replace "may be arbitrary pytree types" with "may be arbitrary pytree values"? (That is, `a` and `b` stand for types while `xs` and `ys` stand for values.)

Reply:
Done, thanks!